notebooks/official/feature_store/online_feature_serving_and_fetching_bigquery_data_with_feature_store.ipynb

```diff
@@ -62,9 +62,9 @@
       "source": [
```

Comment:
<p>Is it intended to expand this list?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2435/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>